src/main/java/edu/harvard/iq/dataverse/DOIDataCiteRegisterService.java

```diff
@@ -189,6 +188,32 @@ public static String getMetadataFromDvObject(String identifier, Map<String, Stri
         return xmlMetadata;
     }
 
+    public static String getMetadataForDeactivateIdentifier(String identifier, Map<String, String> metadata, DvObject dvObject) {
+
+         System.out.print("Start of getMetadataForDeactivateIdentifier: ");
+        DataCiteMetadataTemplate metadataTemplate = new DataCiteMetadataTemplate();
+        metadataTemplate.setIdentifier(identifier.substring(identifier.indexOf(':') + 1));
+        metadataTemplate.setCreators(Util.getListFromStr(metadata.get("datacite.creator")));
+
+        metadataTemplate.setDescription(":unav");
```

Comment:
Where does this special value come from?

Reply:
The special values here come from the DataCite Metadata schema https://support.datacite.org/docs/schema-values-unknown-information-v41

My intent for this is to mask any identifying information since the Dataset in question has been destroyed by Dataverse and in the opinion of the Dataverse admin should never have been created/published.